tests/fast-integration/logging/helpers.js

```diff
@@ -9,8 +9,7 @@ const {
 } = require('./client');
 
 // checkLokiLogs used directly in Commerce Mock tests.
-async function checkLokiLogs(startTimestamp) {
-  const labels = '{app="commerce-mock", container="mock", namespace="mocks"}';
+async function checkLokiLogs(startTimestamp, labels) {
```

Comment:
I think we can also delete the `logpipeline.yaml` since it is not needed anymore

Reply:
Deleted